Cargo.toml

```diff
@@ -133,6 +134,7 @@ axum = { version = "0.6.20", features = [
     "multipart",
     "ws",
 ] } # todo: upgrade this alongside hyper/http/tokio-tungstenite
+axum-server = { version = "0.7.2", features = ["tls-rustls-no-provider"] }
```

Comment:
I think we don't need this?

Reply:
Ah, good catch. I used it earlier on in the build out but didn't end up needing it at the end